src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int bothside;
+    private int height;
+
+    public RightTriangle(String color, int height, int bothside) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. Instead, create separate variables for 'height' and 'bothside', and pass them on for better code readability.